### PR TITLE
Update ar.yml

### PR DESCRIPTION
**Pull request by n-eq**
Add new translations, complete translating 'about' section.

### CODE REVIEW

File: app/assets/javascripts/components/containers/mastodon.jsx [hunk 1]
```diff
@@ -56,6 +56,7 @@ import uk from 'react-intl/locale-data/uk';
 import zh from 'react-intl/locale-data/zh';
 import bg from 'react-intl/locale-data/bg';
 import { localeData as zh_hk } from '../locales/zh-hk';
+import bg from 'react-intl/locale-data/ar';
```
Comment (ykzts) [inline app/assets/javascripts/components/containers/mastodon.jsx]: Is not it `import ar from 'react-intl/locale-data/ar';`?

**Review by ykzts — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/components/containers/mastodon.jsx (hunk 1 above)
Comment (Marrakchino) [inline app/assets/javascripts/components/containers/mastodon.jsx]: That should be it, I suppose. However I'm not the author of the commit in question, I think it's @BoFFire .

Comment (ykzts) [inline app/assets/javascripts/components/containers/mastodon.jsx]: It seems that your commitment and other people 's commit are misleading. This is a slightly strange condition. You should make changes from the current master.

Comment (Marrakchino) [inline app/assets/javascripts/components/containers/mastodon.jsx]: I'll send a patch.

**Comment by ykzts** [comment]
@Marrakchino Your translation (d3289d7d9054928ed27ba5f13d6518f515d2c11a) seems already translated in master ([`./config/locale/ar.yml`](https://github.com/tootsuite/mastodon/blob/b8e166894b1d41ef417c89836a336590d93532f9/config/locales/ar.yml))

File: app/assets/javascripts/components/containers/mastodon.jsx (hunk 1 above)
Comment (BoFFire) [inline app/assets/javascripts/components/containers/mastodon.jsx]: Hum ? Let me check this.

**Comment by Marrakchino** [comment]
I forked an outdated version then, ok.

File: app/assets/javascripts/components/containers/mastodon.jsx (hunk 1 above)
Comment (BoFFire) [inline app/assets/javascripts/components/containers/mastodon.jsx]: @Marrakchino please take file directly from Master branch and not by forking the repo I forked.
A lot of files and strings changed during these last days :)
So please go master then submit patches :)
Thank you.

**Pull request closed without merging**